tests/e2e-test/README.md

```diff
@@ -0,0 +1,30 @@
+# Marinera E2E Test Helm Chart
+
+A Helm chart for Kubernetes to deploy the e2e tests for Marinera repository.
```

Comment:
Shouldn't we say "...to deploy the e2e tests for a FIWARE deployment"? Or sth like that?

Reply:
But will it test every FIWARE deployment? Or just what we are deploying from the marinera repo? For the moment is the second one